roles/openshift_logging/files/generate-jks.sh

```diff
@@ -0,0 +1,21 @@
+#! /bin/sh
+
+set -ex
+
+keytool -genkey -alias {{component}} -keystore {{generated_certs_dir}}/{{component}}.jks -keypass kspass -storepass kspass \
+-keyalg RSA -keysize 2048 -validity 712 -dname "CN={{component}}, OU=OpenShift, O=Logging" {{extensions.stdout}}
+
+keytool -certreq -alias {{component}} -keystore {{generated_certs_dir}}/{{component}}.jks -storepass kspass \
+-file {{generated_certs_dir}}/{{component}}-jks.csr -keyalg RSA -dname "CN={{component}}, OU=OpenShift, O=Logging" {{extensions.stdout}}
+
+# this currently fails since we don't provide openssl as part of the ES image...
+openssl ca -in {{generated_certs_dir}}/{{component}}-jks.csr -notext -out {{generated_certs_dir}}/{{component}}-jks.crt \
+-config {{generated_certs_dir}}/signing.conf -extensions v3_req -batch -extensions server_ext
```

Comment:
the alternative work was to use PKCS12, however Searchguard didn't accept any of the cert chains created with openssl. Their documentation only gives examples creating using keytool (jks or pkcs12).

@detiber i hadn't tried to do anything like the former, but I think there are some oddities with generating the csr/key. I'll see if it'd be possible to gen them with the OpenShift CA and then import them into a JKS.